packages/wp-story-editor/src/components/documentPane/index.js

```diff
@@ -13,5 +13,31 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/**
+ * External dependencies
+ */
+import {
+  PublishPanel,
+  ExcerptPanel,
+  SlugPanel,
+  StatusPanel,
+  PageAdvancementPanel,
+  BackgroundAudioPanel,
+  TaxonomiesPanel,
+} from '@web-stories-wp/story-editor';
+
+function DocumentPane() {
+  return (
+    <>
+      <StatusPanel />
```

Comment:
We should move this component to `wp-story-editor`

It already contains lots of WP specific references (e.g. "Visible to site admins & editors only").
Plus, #9502 is going to update it to accommodate for WP specific quirks (e.g. WP does not supports private drafts, so private means instantly publishing the story).

Reply:
Yes, makes sense 👍